packages/cross_file/lib/src/implementations/io_x_file.dart

```diff
@@ -0,0 +1,86 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+import 'dart:typed_data';
+
+import '../x_file.dart';
+
+/// A CrossFile backed by a dart:io [File].
+///
+/// (Mobile-only).
```

Comment:
Don't forget desktop :)

```suggestion
/// This is only supported on non-web platforms.
```